### DISCUSSION TITLE
Send trivy scans to Postee?

### DISCUSSION THREAD

**Discussion by kuwaharg** [aquasecurity/trivy-operator#366] [Help & Support] [answered]
Can you send trivy-operator scans to Postee?

We have trivy-operator installed in our cluster via helm. We would like to send messages to our teams channel when a critical vulnerability is detected. Most of the directions I can find show using an Aqua console for configuration. We do not have an Aqua console for our setup.

Is there another way to achieve this?

Thanks! --Gretchen

(I have also posted this question on the postee discussions board)

**Comment by chen-keinan** [comment, marked as answer]
@kuwaharg we have an Item for it #70 @simar7 FYI.

**Reply by kuwaharg** [reply]
Awesome!  I'll keep an eye on that issue.

Thanks for the quick response! --Gretchen

**Comment by simar7** [comment]
hi @kuwaharg - https://github.com/aquasecurity/trivy-operator/pull/476 was merged and will be available as a feature in the next release of Trivy operator.
